tasks/create.yml

```diff
@@ -51,20 +59,25 @@
 
 - name: Stop and delete server and client service
   block:
+  - name: Probe installed services
```

Comment:
What’s this?

Reply:
The issue I encountered was that when we have the force flag set, but BeeGFS packages are not yet installed, the systemd service shutdown will fail.  Here I added a test to check for the presence of the service (which should be portable for modern distros), and it is conditionally disabled only if the service exists in the first place.

An alternative would be to allow the service shutdown task to fail, but I think has drawbacks.